docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -308,6 +308,12 @@ Tip: The ``group_vars/`` and ``host_vars/`` directories can exist in
 the playbook directory OR the inventory directory. If both paths exist, variables in the playbook
 directory will override variables set in the inventory directory.
 
+Tip: When running the ``ansible`` command (rather than
```

Comment:
missing ``ansible-console``

Reply:
Does 317591558653795e32f20009f233d9c82f0eb3c0 retain the original intent of the PR and address your concerns @bcoca?